packages/common-cosmos/src/project/project.spec.ts

```diff
@@ -32,4 +32,15 @@ describe('project.yaml', () => {
     expect(validateCosmosManifest(cosmosManifest)).toBe(true);
     expect(validateCosmosManifest(ethManifest)).toBe(false);
   });
+  it('Should fail on incorrect chaintypes', () => {
+    const cosmosManifest = loadFromJsonOrYaml(
+      path.join(projectsDir, './protoTest1', 'bad-chaintypes-project.yaml')
+    ) as any;
+    expect(() => parseCosmosProjectManifest(cosmosManifest)).toThrow('failed to parse project.yaml');
+  });
+  it('Ensure correctness on manifest deployment', () => {
```

Comment:
Maybe we can be more specific with this name as we only validate the chain types are included